feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/ui/CommonClaims.kt

```diff
@@ -58,7 +59,7 @@ internal fun CommonClaims(
             ) {
               Text(claim.displayName, Modifier.align(Alignment.CenterVertically))
               Icon(
-                painter = painterResource(id = com.hedvig.android.odyssey.R.drawable.ic_list_item_indicator),
+                painter = painterResource(id = R.drawable.ic_arrow_forward),
```

Comment:
Is this any different from the one which comes from `Icons.Default.ArrowForward`?
I think we should just use that directly, as it's for sure the one which is what's expected for android, as seen in [here](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:arrow_forward:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=arrow+for) which is `Icons.Default.ArrowForward` in compose.
This way we also can remove the drawable XML completely.

Reply:
Excellent, will change to using that instead 👍